### PR TITLE
[FX-2245] ReadMore styling

### PR DESCRIPTION
**Pull request by dzucconi**
Re: [FX-2245](https://artsyproduct.atlassian.net/browse/FX-2245)

* Some minor updates to `Clickable` to make it easier to use
* Updates `HTML` to export it's default styling
* Utilizes `HTML` styling on `ReadMore`

<details>
  <summary>📦 Published PR as canary version: <code>13.6.1-canary.782.13312.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```sh
  npm install @artsy/palette@13.6.1-canary.782.13312.0
  # or 
  yarn add @artsy/palette@13.6.1-canary.782.13312.0
  ```
</details>

### CODE REVIEW

**Comment by dzucconi** [comment]
Worth noting we can mostly get away with not touching ReadMore, if it's desirable for that to not have any opinions about styling and just function as truncation. — I'm operating under the assumption here that it should have some default opinions about it's content.

File: packages/palette/src/elements/ReadMore/ReadMore.tsx [hunk 1]
```diff
@@ -71,59 +74,45 @@ export class ReadMore extends Component<ReadMoreProps, ReadMoreState> {
...
     )
   }
 }
 
-const ReadMoreLink = ({ children }) => {
+const ReadMoreLink: React.FC<ClickableProps> = ({ children, ...rest }) => {
   return (
-    <span>
+    <>
       {" "}
```
Comment (damassi) [inline packages/palette/src/elements/ReadMore/ReadMore.tsx:98]: Is this space needed?

File: packages/palette/src/elements/ReadMore/__tests__/ReadMore.test.tsx [hunk 2]
```diff
@@ -31,11 +31,11 @@ describe("ReadMore", () => {
     expect(wrapper.find("ReadMoreLink").length).toEqual(0)
   })
 
-  it("expands text on click", () => {
+  fit("expands text on click", () => {
```
Comment (damassi) [inline packages/palette/src/elements/ReadMore/__tests__/ReadMore.test.tsx:34]: focused

File: packages/palette/src/elements/ReadMore/ReadMore.tsx (hunk 1 above)
Comment (dzucconi) [reply packages/palette/src/elements/ReadMore/ReadMore.tsx:98]: Tis

File: packages/palette/src/elements/ReadMore/__tests__/ReadMore.test.tsx (hunk 2 above)
Comment (dzucconi) [reply packages/palette/src/elements/ReadMore/__tests__/ReadMore.test.tsx:34]: 🤦

**Comment by dzucconi** [comment]
I'm going to make this a bit simpler and roll off the default styling changes.

**Comment by dzucconi** [comment]
OK, bit of a lighter touch here: I've walked back automatically styling the contents of `ReadMore` with `HTML` by default, which I think is a bit presumptuous — instead `HTML` can accept either an `html` string or `children` and style it accordingly, so wrapping `ReadMore` with `HTML` will work, as will wrapping it in anything else.

**Comment by dzucconi** [comment]
Going to merge on 🍏  but feel free to leave any feedback for me to address.

**Comment by artsyit** [comment]
:rocket: PR was released in [`v13.7.0`](https://github.com/artsy/palette/releases/tag/v13.7.0) :rocket:

**Pull request merged**